site/content/en/docs/tasks/manage/installation/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Configuration Installation of Kueue"
```

Comment:
```suggestion
title: "Configure external cert-manager"
```
I suggest to focus the new pare on cert-management only. We can later group some of the pages under 
"Productizing aspects of Kueue" as proposed in https://github.com/kubernetes-sigs/kueue/issues/3087

Reply:
Or alternatively, feel free to rename it as "Setup for productization"?

Reply:
Is that under manage?

Reply:
I put this under tasks/manage/producization.